packages/fetchye/__tests__/computeKey.spec.js

```diff
@@ -58,4 +67,55 @@ describe('computeKey', () => {
     };
     expect(computeKey('uri', firstOptions).hash).toBe(computeKey('uri', secondOptions).hash);
   });
+
+  it('should return a different, stable hash, if the option mapKeyToCacheKey is passed', () => {
+    expect(computeKey(() => 'abcd', {
+      mapKeyToCacheKey: () => 'efgh',
+    })).toMatchInlineSnapshot(`
+      Object {
+        "hash": "a0e09d568bb5b47c046b0fac7a61ca10196151cc",
+        "key": "abcd",
+      }
+    `);
+  });
+
+  it('should pass generated cacheKey to the underlying hash function along with the options, and return the un-mapped key to the caller', () => {
+    const computedKey = computeKey(() => 'abcd', {
+      mapKeyToCacheKey: (key, options => `${key.toUpperCase()}-${options.optionKeyMock}`,
+      optionKeyMock: 'optionKeyValue',
+    });
+    expect(computedKey.key).toBe('abcd');
+    expect(computeHash).toHaveBeenCalledWith(['ABCD-optionKeyValue', { optionKeyMock: 'optionKeyValue' }], { respectType: false });
+  });
+
+  it('should return the same key if the option mapKeyToCacheKey returns the same string as the key', () => {
```

Comment:
This test has the same issue as the one on line 71

Reply:
addressed in a693a07